api/src/main/java/com/codeforcommunity/dto/team/TeamDataResponse.java

```diff
@@ -0,0 +1,67 @@
+package com.codeforcommunity.dto.team;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class TeamDataResponse {
+  private final Integer id;
+  private final String teamName;
```

Comment:
There's usages of lowerCamelCase and snake_case throughout different files. Which is more prevalent in this codebase? I personally prefer lowerCamelCase as that's what we use on the frontend.

Reply:
lowerCamelCase is mostly used. I prefer this too for names of Java Variables. There are some instances when dealing with input from JSON where snake_case is used. I cannot find any instances of snake_case that are unnecessary.